utils/src/test/kotlin/storage/FileArchiverTest.kt

```diff
@@ -111,5 +111,18 @@ class FileArchiverTest : StringSpec() {
                 assertFileContent("c/b")
             }
         }
+
+        "LICENSE files are archived by default, independently of the directory" {
+            createFile("LICENSE")
+            createFile("path/LICENSE")
+
+            FileArchiver.DEFAULT.archive(workingDir, "save")
+            FileArchiver.DEFAULT_ARCHIVE_DIR.resolve("save/archive.zip").unpack(targetDir)
```

Comment:
This way the test is using the `~/.ort` directory, it should use a temporary directory instead, like the other tests.

Reply:
Done.